types/sidebar.ts

```diff
@@ -0,0 +1,14 @@
+import type { Url } from 'url';
+
+export interface SidebarItemType {
+  url: string | Url;
+  title: string;
+}
+
+export interface SidebarGroupType {
+  groupName: string;
```

Comment:
```suggestion
  name: string;
```

IMO, it is a part of `SidebarGroupType`, we do not need to name it as a `groupX` again below

Reply:
I have followed example schema described on issue #6065 [
  {
     groupName: "Getting Started",
     items: [
        {
          url: "",
          title: ""
        },
        ...
     ]
  },
  ...
]